test/starknet/ControllerActions.test.ts

```diff
@@ -48,4 +49,54 @@ describe('Controller Actions', () => {
       to_remove: votingStrategies,
     });
   }).timeout(600000);
+
+  it('The controller can add and remove authenticators', async () => {
+    const authenticators: bigint[] = [BigInt(1234), BigInt(4567)];
+    await controller.invoke(space, 'add_authenticators', {
+      to_add: authenticators,
+    });
+    await controller.invoke(space, 'remove_authenticators', {
+      to_remove: authenticators,
+    });
```

Comment:
We're checking if adding / removing doesn't throw, but we're not checking if we can then call the space with a different authenticator. Meaning if the function `add_authenticators` was empty, we wouldn't know it from this test :p